about.html

```diff
@@ -0,0 +1,221 @@
+        <div class="about">
+          <h4>73rd FIFA Congress Meeting</h4>
+          <h2>
+            CREATIVE COMMONS<br />
+            GLOBAL SUMMIT 2023
+          </h2>
+          <hr class="line4" />
+        </div>
+        <div class="welcome-about2">
+          <h1>"Hello!Sharing world"</h1>
+          <h2>
+            CREATIVE COMMONS<br />
+            GLOBAL SUMMIT 2023
+          </h2>
+        </div>
+
+        <div class="texts">
+          <p class="p3">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years.
+          </p>
+
+          <p class="p4">
+            This year, we hope to expand our invitation list including
+            organizations and individuals who want to work with us on shared
+            projects that advance the cause of the Commons, free culture and
+            open knowledge.
+          </p>
+          <p class="p5">
+            So if you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+        </div>
+      </div>
+
+      <div class="texts2">
+        <p class="p3">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Commons affiliate network in a
+          different country every two years. This year, we hope to expand our
+          invitation list including organizations and individuals.
+        </p>
+      </div>
+      <div class="about-contact">
+        <p>
+          Please contact us per Email for nay further questions about CC Global
+          Summit 2023!
+        </p>
+        <h4>ccsummit2023@cckorea.org</h4>
+      </div>
+    </div>
+    <div class="about-section">
+      <div class="container7">
+        <h4>
+          Creative Commons<br />
+          Global Summit 2023 Logo
+        </h4>
+        <hr class="line4" />
+        <p class="p6">
+          The logo of CC Global Summit 2023 was decided through the logo
+          competition from 8, June to 7, July.
+        </p>
+        <p class="p7">
+          The logo of CC Global Summit 2023 was decided through the logo
+          competition from 8, June to 7, July. <br />After online public voting
+          and Selection Committee’s judging process, the logo from Naresh
+          Agrawal won the prize.<br />
+          After consultation with Mr. Agrawal, we optimized the
+          prize-winning-logo for this year’s summit as follows:
+        </p>
+        <img class="logo-about" src="images/about_logo 1.png" />
```

Comment:
- [ ] 🙏Please adjust the box size of this image on the mobile screen📱 to create space on the right and left sides, resembling the original design. This will enhance the appearance🌹 of your design and make it as cool as the original.

Your design:
![image](https://github.com/meryemsanem/capstone-project/assets/63445083/6d45c1a0-6837-42ec-8039-25474316bf7e)

Original design:
![image](https://github.com/meryemsanem/capstone-project/assets/63445083/ea81e000-fbc6-41a1-8c44-18d67b70d3f2)